xray/test/test_plotting.py

```diff
@@ -201,7 +208,10 @@ def test_xyincrease_true_changes_axes(self):
 
     def test_plot_nans(self):
         self.darray[0, 0] = np.nan
-        self.plotmethod()
+        result = self.plotmethod()
+        clim = result.get_clim()
+        self.assertTrue(clim[0] <= -2)
+        self.assertTrue(clim[1] >= 2)
```

Comment:
I'm seeing values outside this range without nan's. Maybe more clear to say `self.assertFalse(any(np.isnan(clim)))`?

Thanks for improving this test btw.